code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1518,3 +1518,240 @@
+	name = "Slime Mold"
+	id = "slimemold"
+	description = "You can swear that this jelly looks alive."
+	color = "#C20458" // rgb: 194, 4, 88
+	alcohol_perc = 0.2
+	drink_icon = "slimemold"
+	drink_name = "Slime Mold"
+	drink_desc = "You can swear that this jelly looks alive."
+	taste_description = "jelly"
+
+/datum/reagent/consumable/ethanol/slime_mold/on_mob_life(mob/living/M)
+	if(isslimeperson(M))
+		var/mob/living/carbon/human/H = M
+		if(!(NO_BLOOD in H.dna.species.species_traits))
+			if(H.blood_volume < BLOOD_VOLUME_NORMAL)
+				H.blood_volume += REAGENTS_METABOLISM / 2 // half of the reagent is converted into blood, netting us just a little bit
+	return ..()
+
+/datum/reagent/consumable/ethanol/die_seife
+	name = "Die Seife"
+	id = "dieseife"
+	description = "There is a piece of soap at the bottom of the glass and it is slowly melting."
+	color = "#9D9E89" // rgb: 157, 158, 137
+	alcohol_perc = 0.2
+	drink_icon = "dieseife"
+	drink_name = "Die Seife"
+	drink_desc = "There is a piece of soap at the bottom of the glass and it is slowly melting."
+	taste_description = "soap"
+
+/datum/reagent/consumable/ethanol/die_seife/on_mob_life(mob/living/M)
+	if(isdrask(M))
+		if(current_cycle % 10 == 0 && prob(50))
+			to_chat(M, "<span class='warning'>Your skin emits a soapy liquid from its pores cleaning you in the process.</span>")
+			M.clean_blood()
+	return ..()
+
+/datum/reagent/consumable/ethanol/acid_dreams
+	name = "Acid Dreams"
+	id = "aciddreams"
+	description = "This one looks just wierd and reeks of acid."
+	color = "#B7FF6A" // rgb: 183, 255, 106
+	alcohol_perc = 0.7
+	drink_icon = "aciddreams"
+	drink_name = "Acid Dreams"
+	drink_desc = "This one looks just wierd and reeks of acid."
+	taste_description = "acid"
+
+/datum/reagent/consumable/ethanol/acid_dreams/on_mob_life(mob/living/M)
+	if(isgrey(M))
+		if(current_cycle % 10 == 0 && prob(50))
+			var/list/mob/living/targets = list()
+			for(var/mob/living/L in orange(14, M))
+				if(L.is_dead()) //we don't care about dead mobs
+					continue
+				if(!L.client)
+					continue
+				targets += L
+			var/mob/living/target = pick(targets)
+			if(target == null)
+				return ..()
+			to_chat(M, "<span class='warning'>You feel that [target.name] is somewhere near.</span>")
+
+	return ..()
+
+/datum/reagent/consumable/ethanol/islay_whiskey
+	name = "Islay Whiskey"
+	id = "islaywhiskey"
+	description = "Named in honor of one of the most gritty and earth smelling types of Whiskey of Earth, this drink is a treat for any Diona."
+	color = "#461300" // rgb: 70, 19, 0
+	alcohol_perc = 0.2
+	drink_icon = "islaywhiskey"
+	drink_name = "Islay Whiskey"
+	drink_desc = "Named in honor of one of the most gritty and earth smelling types of Whiskey of Earth, this drink is a treat for any Diona."
+	taste_description = "soil"
+
+/datum/reagent/consumable/ethanol/islay_whiskey/on_mob_life(mob/living/M)
+	if(isdiona(M))
+		if(current_cycle > 5)
+			var/mob/living/carbon/human/H = M
+			var/turf/T = H.loc
```

Comment:
```suggestion
			var/turf/T = get_turf(H)
```
loc will not always be a turf, it could a locker, mob, container, etc. Wherever the mob is located at.

Reply:
updated